packages/util/src/units.ts

```diff
@@ -0,0 +1,118 @@
+// Source of constants: https://www.languagesandnumbers.com/articles/en/ethereum-ether-units/
+export enum Unit {
+  Wei = 'wei',
+  Kwei = 'kwei',
+  Mwei = 'mwei',
+  Gwei = 'gwei',
+  Micro = 'micro',
+  Milli = 'milli',
+  Ether = 'ether',
+  Kether = 'kether',
+  Megaether = 'megaether',
+  Gether = 'gether',
+}
+
+export enum UnitValue {
+  'wei' = -18,
+  'kwei' = -15,
+  'mwei' = -12,
+  'gwei' = -9,
+  'micro' = -6,
+  'milli' = -3,
+  'ether' = 1,
+  'kether' = 3,
+  'megaether' = 6,
+  'gether' = 9,
+  'tether' = 12,
+}
+export type UnitsNames = keyof typeof UnitValue
```

Comment:
would suggest singular here

```suggestion
export type UnitNames = keyof typeof UnitValue
```